usb/usb_host_vcp/idf_component.yml

```diff
@@ -1,9 +1,13 @@
 ## IDF Component Manager Manifest File
-version: "1.0.0~1"
+version: "1.0.0~2"
 description: USB Host Virtual COM Port Service
 url: https://github.com/espressif/idf-extra-components/tree/master/usb/usb_host_vcp
 dependencies:
   espressif/usb_host_cdc_acm:
     version: ">=1.0.4,<3.0.0"
     public: true
   idf: ">=4.4"
+targets:
+  - esp32s2
+  - esp32s3
+  - esp32p4
```

Comment:
Same here, the component depends on `usb_host_cdc_acm` so the target requirements are taken from there